plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1550,13 +1550,17 @@ private void checkColumnTypes(List<ColumnMetadata> columnMetadata)
         }
     }
 
-    private static void deleteRecursivelyIfExists(TrinoFileSystem fileSystem, Location path)
+    private static void deleteRecursivelyIfLogNotExists(TrinoFileSystem fileSystem, Location tablePath)
     {
         try {
-            fileSystem.deleteDirectory(path);
+            Location deltaLogDirectory = Location.of(getTransactionLogDir(tablePath.path()));
```

Comment:
This code throws exception which is only logged:
```
2024-12-18T06:58:49.952-0600	ERROR	transaction-finishing-0	io.trino.metadata.CatalogMetadata	Connector threw exception on abort
java.lang.IllegalArgumentException: No scheme for file system location: var/folders/ns/drnmv_551mjcn3rd0ccw1tzm0000gn/T/catalog-dir5122020563593486373/tpch/test_concurrent_ctas_vnae2mtwc9/_delta_log
```
It causes probably CI failures.
It also visible in logs for `testConcurrentCreateTableAsSelectSameTable`